EmotiBit.cpp

```
@@ -3707,72 +3762,140 @@ EmotiBit::PowerMode EmotiBit::getPowerMode()
+		}
+	}
+	else if (mode == PowerMode::HIBERNATE)
 	{
 		Serial.println("PowerMode::HIBERNATE");
+		_powerMode = mode;
 	}
 	else
 	{
 		Serial.println("PowerMode Not Recognized");
 	}
+	String modePacket;
```

Clang (14.0.6)

[warning] variable 'modePacket' is not initialized

(cppcoreguidelines-init-variables)